internal/server/http/handler/monitoring.go

```diff
@@ -0,0 +1,26 @@
+package handler
+
+import (
+	"net/http"
+
+	"github.com/gol4ng/logger"
+)
+
+type Liveness struct {
+	logger logger.LoggerInterface
+}
+
+// NewLiveness returns a liveness HTTP request handler
+func NewLiveness(logger logger.LoggerInterface) http.Handler {
+	return Liveness{logger: logger}
+}
+
+// ServeHTTP handles an HTTP request
+func (h Liveness) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	if r.Method != "GET" {
+		respond(w, errorJSON("only GET requests are supported"), http.StatusMethodNotAllowed)
```

Comment:
it should be http.StatusBadRequest instead

Reply:
Here we check the HTTP method so `http.StatusMethodNotAllowed` looks more accurate I think

Reply:
we can move the allowed method directly in the route declaration in the router